src/yea/schema-yea.json

```diff
@@ -88,6 +94,18 @@
                         "type": "string"
                     }
                 },
+                "pip_install_options": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "uninstall": {
```

Comment:
don't you also need to add the function in `ytest` that runs uninstall?

Reply:
I was going to leave that for you... :)

Reply:
oh sure :)